tests/src/Unit/Page/PageAttachmentsEventTest.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Drupal\Tests\hook_event_dispatcher\Unit\Page;
+
+use Drupal\Core\DependencyInjection\ContainerBuilder;
+use Drupal\hook_event_dispatcher\Event\Page\PageAttachments;
```

Comment:
Please update this use statement as well, so the Event can be used 😉

Reply:
🤦‍♀ Can't believe I missed that. Thanks for your patience.